diplomacy/web/src/gui/pages/content_game.jsx

```diff
@@ -1816,7 +1820,6 @@ export class ContentGame extends React.Component {
         const receivedSuggestions = globalMessages.filter(
             (msg) =>
                 msg.type === STRINGS.SUGGESTED_COMMENTARY &&
-                msg.parsed.payload.recipient === protagonist &&
```

Comment:
This line is needed to ensure a player isn't shown commentary advice meant for someone else. Is there a reason you removed it?